### ISSUE TITLE
Flutter Syncfusion DataGridView Flicker when scroll bar

### ISSUE THREAD

**Issue by abdulrehmananwar** [syncfusion/flutter-widgets#1301]
i already create this issue earlier. this happends only some times. that grid white expect current row. this issue only happend when scoll bar is started . and happend only some times. and when i press tab key. please see in attached video

https://github.com/syncfusion/flutter-widgets/assets/62530396/7f6212c7-bf78-4332-844d-63685add3a21

**Comment by Tamilarasan-Paranthaman** [comment]
Hi @abdulrehmananwar,

Based on the information provided, we have conducted tests on our end, but we were unable to reproduce the reported issue of the DataGrid flickering when the scroll bar is displayed. It seems that the DataGrid is functioning correctly without any flickering.

To better assist you and understand the problem you are facing, modify the attached simple sample as reproducible. By having a reproducible sample, we will be able to thoroughly investigate the problem and offer a solution in a timely manner.

**Sample Link: [Sample](https://github.com/syncfusion/flutter-widgets/files/12069691/sample.zip)**


https://github.com/syncfusion/flutter-widgets/assets/93904422/dc5c164e-83db-48c2-8184-8b2213f72ade


Note: We have tested this sample with the latest DataGrid version 22.1.38.

Regards,
Tamilarasan
